### ISSUE TITLE
[FEATURE] Support Linux

### ISSUE THREAD

**Issue by tsilvs** [vitor251093/resurrection-capsule#11]
# Describe the solution you'd like
I'd like to be able to host the server on a Linux machine, as well as running the game through Wine / Proton with the request redirection patch.

# Describe alternatives you've considered
* A guide to building the project from source on Linux
* Linux included as target platform for builds

**Comment by vitor251093** [comment]
I've bought a new computer recently, and it should be with me in about 2 weeks.

I will be installing Ubuntu in that new computer, so I should be able to work on that with it. I'm more eager to work in both alternatives, since they have more long term applications.

**Comment by JeffRockatansky** [comment]
@vitor251093 
> I will be installing Ubuntu in that new computer

Congrats on it! Welcome to the community! I also recommend you to try Manjaro, Kubuntu & Pop.

**Comment by vitor251093** [comment]
> I also recommend you to try Manjaro, Kubuntu & Pop.
I tried Pop last year, but the system update broke the system startup lol

> Different distos may vary in details, so your project will only benefit from being more platform-neutral. So don't focus on Ubuntu-specific stuff.
Indeed. At first, I'm focusing in distributing a Debian build, so it should be compatible with a wide range of distros out of the gate.

**Comment by JeffRockatansky** [comment]
> I tried Pop last year, but the system update broke the system startup lol

Yeah, it might be like that sometimes. That's why it's the last in my list. Maybe I should've skipped it at all.

**Comment by vitor251093** [comment]
> Don't forget about Pacman and AppImages too. 

Could be. That's just to start from somewhere ;)

> You could start from Arch package format & makepkg and then create .debs using makedeb. I myself successfully built some apps from AUR with it.

I have no idea of how to do that. Do know any guide I could follow to do that?

**Comment by JeffRockatansky** [comment]
> any guide I could follow to do that?

[README](https://github.com/makedeb/makedeb/blob/stable/README.md) and [official docs](https://makedeb.hunterwittenborn.com/home/introduction/) are enough to start.

**Comment by all500234765** [comment]
I would actually be against that due to the fact that linux support always requires way too much effort for marginal gains.
I'd suggest supporting a single OS for the time being. If linux users would want to, they should port server them selves instead of asking a single dev working in his spare time to basically _kill time_ supporting linux.

**Comment by NovaViper** [comment]
I would suggest making a Flatpak version instead of direct package (like Debian), at least for a start. Flatpak would allow the game to run on multiple different distros without having to spend a lot of packaging for each and every different Linux distro.

**Comment by vitor251093** [comment]
Done, and currently available at vitor251093/recap_server

It will be included on the next official release
